common/src/IO/AssimpParser.cpp

```diff
@@ -350,8 +481,77 @@ void processNode(
   }
 }
 
+static const auto AiMdlHl1NodeBodyparts = "<MDL_bodyparts>";
+
+void processRootNode(
+  std::vector<AssimpMeshWithTransforms>& meshes,
+  const aiNode& node,
+  const aiScene& scene,
+  const aiMatrix4x4& transform,
+  const aiMatrix4x4& axisTransform)
+{
+  // HL1 models have a slightly different structure than normal, the format consists
+  // of multiple body parts, and each body part has one or more submodels. Only one
+  // submodel per body part should be rendered at a time.
+
+  // See if we have loaded a HL1 model
+  if (const auto hl1BodyParts = node.FindNode(AiMdlHl1NodeBodyparts))
+  {
+    // HL models are loaded by assimp in a particular way, each bodypart and all
+    // its submodels are loaded into different nodes in the scene. To properly
+    // display the model, we must choose EXACTLY ONE submodel from each body
+    // part and render the meshes for those chosen submodels.
+
+    // Assimp HL models face sideways by default so spin by -90 on the z axis
+    // this MIGHT be needed for non-HL models as well. To be safe for now, we
+    // only do this for HL models.
+    const aiQuaternion rotation(aiVector3t(0, 1, 0), -vm::Cf::half_pi());
```

Comment:
Change to `const auto rotation = aiQuaternion{aiVector3t{0, 1, 0}, -vm::Cf::half_pi()};`